aptos-move/aptos-release-builder/data/release.yaml

```diff
@@ -1,24 +1,14 @@
 ---
 remote_endpoint: ~
-name: "v1.13"
+name: "v1.14"
 proposals:
-  - name: step_1_upgrade_gas_schedule
+  - name: step_1_upgrade_framework
     metadata:
-      title: "Gas schedule upgrade"
-      description: "Upgrades the gas schedule to version 18, enabling separate limits for governance proposals"
-    execution_mode: MultiStep
-    update_sequence:
-      - DefaultGasWithOverride:
-          feature_version: 17
-          overrides:
-            - name: "txn.max_execution_gas"
-              value: 4000000000
-  - name: step_2_upgrade_framework
-    metadata:
-      title: "Multi-step proposal to upgrade mainnet framework to v1.13"
+      title: "Multi-step proposal to upgrade mainnet framework to v1.14"
       description: "This includes changes in https://github.com/aptos-labs/aptos-core/commits/aptos-release-v1.13"
```

Comment:
"v1.13" -> "v1.14"?

Reply:
1.14 branch does not exist yet, we can update the description later